pkg/gateway/preview_proxy_test.go

```
@@ -317,13 +317,74 @@ func TestPreviewProxy_InjectsBrowserOverlayIntoHTML(t *testing.T) {
+	secondRoute := f.seed(t, "bob", tokens.VisibilityPublic)
+
+	acknowledge, err := http.NewRequest(http.MethodPost, f.srv.URL+webpreview.LauncherSeenPath, nil)
+	if err != nil {
+		t.Fatalf("NewRequest: %v", err)
+	}
+	acknowledge.Host = tokens.HostFor(firstRoute.Token, f.root)
+	acknowledged, err := http.DefaultClient.Do(acknowledge)
+	if err != nil {
+		t.Fatalf("acknowledge launcher: %v", err)
+	}
+	defer acknowledged.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `acknowledged.Body.Close` is not checked

(errcheck)